src/foundry/client/data/documents/actor.d.mts

```diff
@@ -62,6 +66,11 @@ declare global {
       TokenDocument.ConfiguredInstance
     >;
 
+    protected override _initializeSource(
+      data: this | SchemaField.InnerAssignmentType<DataSchema>,
+      options?: any,
```

Comment:
Please make this `unknown`. The difference isn't critical but still. I think I type this more strongly in my branch so I won't be a stickler to get it into its proper object type though.